variables.tf

```diff
@@ -134,3 +134,163 @@ variable "alias" {
+}
+
+variable "policy_scale_down_interval_lower_bound" {
+  description = "Difference between the alarm threshold and the CloudWatch metric."
+  default     = "0"
+}
+
+variable "policy_scale_down_adjustment" {
+  description = "Nomber of tasks to remove for scaling down."
+  default     = "-1"
+}*/
+/* ==== END AUTOSCALE POLICY SCALEDOWN VARIABLES ==== */
+
+/* ========= BEGIN COMMON CLOUDWATCH METRIC ALARM VARIABLES ==== */
+/*variable "metric_name" {
+  description = "Name of the Metric"
+  default     = "MemoryUtilization"  
+} 
+variable "datapoints_to_alarm" {
+  description = "No. of times the metric reaches the specified threshold to trigger an alarm."
+  default     = "3"
+}*/
+/* ========= END COMMON CLOUDWATCH METRIC ALARM VARIABLES ==== */
+
+/* ==== BEGIN CLOUDWATCH METRIC ALARM SCALEUP VARIABLES ==== */ 
+/*variable "scale_up_comparison_operator" {
+  description = "The arithmetic operation to use when comparing the specified Statistic and Threshold. The specified Statistic value is used as the first operand (for ex: GreaterThanOrEqualToThreshold, GreaterThanThreshold, LessThanThreshold, LessThanOrEqualToThreshold)"
+  default     = "GreaterThanOrEqualToThreshold"
+}
+
+variable "scale_up_evaluation_periods" {
+  description = "The number of periods over which data is compared to the specified threshold."
+  default     = "1"
+}
+variable "scale_up_period" {
+  description = "The period in seconds over which the specified statistic is applied."
+  default     = "60"
+}
+variable "scale_up_statistic" {
+  description = "The statistic to apply to the alarm's associated metric."
+  default     = "Sum"
+}
+variable "scale_up_threshold" {
+  description = "The value against which the specified statistic is compared."
+  default     = "70"
+}*/
+
+/* ==== END CLOUDWATCH METRIC ALARM SCALEUP VARIABLES ==== */
+
+/* ==== BEGIN CLOUDWATCH METRIC ALARM SCALEDOWN VARIABLES ==== */
+/*variable "scale_down_comparison_operator" {
+  description = "The arithmetic operation to use when comparing the specified Statistic and Threshold. The specified Statistic value is used as the first operand (for ex: GreaterThanOrEqualToThreshold, GreaterThanThreshold, LessThanThreshold, LessThanOrEqualToThreshold)"
+  default     = "LessThanThreshold"
+}
+variable "scale_down_evaluation_periods" {
+  description = "The number of periods over which data is compared to the specified threshold."
+  default     = "1"
+}
+variable "scale_down_period" {
+  description = "The period in seconds over which the specified statistic is applied."
+  default     = "60"
+}
+variable "scale_down_statistic" {
+  description = "The statistic to apply to the alarm's associated metric."
+  default     = "Average"
+}
+variable "scale_down_threshold" {
+  description = "The value against which the specified statistic is compared."
+  default     = "75"
+}*/
+/* ==== END CLOUDWATCH METRIC ALARM SCALEDOWN VARIABLES ==== */
+
+/* ==== BEGIN SNS NOTIFICATION VARIABLES ==== */
```

Comment:
Do we want to capture this for use in the future, or should we remove commented-out lines?

Reply:
I am uploading a copy of variables.tf and the terraform.tf (SNS notifications code) into the Box folders so that we can use it in future.